hikari/events/guild_events.py

```diff
@@ -390,6 +397,12 @@ class EmojisUpdateEvent(GuildEvent):
     guild_id: snowflakes.Snowflake = attr.ib()
     # <<inherited docstring from GuildEvent>>.
 
+    old_emojis: typing.Optional[cache.CacheView[snowflakes.Snowflake, emojis_.KnownCustomEmoji]] = attr.ib()
```

Comment:
For consistency this should not be a CacheView. You should make this a Mapping instead (will require changing when its passed here)